packages/core-utils/package.json

```diff
@@ -13,7 +13,7 @@
   "private": false,
   "dependencies": {
     "@mapbox/polyline": "^1.1.0",
-    "@opentripplanner/geocoder": "^1.1.0",
+    "@opentripplanner/geocoder": "^1.1.1",
```

Comment:
Latest is now 1.1.2.

Reply:
upgraded in adf39ed79fe9ef36db930bb0c93555320077dc92